src/lgr_models/models.py

```diff
@@ -32,7 +35,18 @@ def filename(self):
 
 
 class RzLgr(LgrModel):
-    pass
+    def __str__(self):
+        return u'{0}'.format(self.name)
+
+    def to_lgr_info(self) -> LGRInfo:
+        self.file.seek(0)
+        return LGRInfo.from_dict(
+            {
+                'name': self.name,
+                'xml': self.file.read(),
+                'lgr': LGR(self.name),
```

Comment:
do you need lgr parameter?

Reply:
No for my use cases, but isn't the main member of LGRInfo? I was finding silly to not add it (and surprised It works nevertheless).